asmcli/components/control-plane/managed.sh

```diff
@@ -33,10 +33,16 @@ install_managed_control_plane() {
     --header "Content-Type: application/json" \
     -K <(auth_header "$(get_auth_token)")
 
-  local VALIDATION_URL; local CLOUDRUN_ADDR;
-  read -r VALIDATION_URL CLOUDRUN_ADDR <<EOF
-$(scrape_managed_urls)
```

Comment:
Had to move this out of `read`, because putting it in an "EOF" block eats the `fatal` call for some reason?